src/sentry/static/sentry/app/views/installWizard.jsx

```diff
@@ -61,11 +61,16 @@ export default class InstallWizard extends AsyncView {
     const data = {};
     Object.keys(options).forEach(optionName => {
       const option = options[optionName];
+      if (option.field.disabled) {
+        return;
+      }
       // XXX(dcramer): we need the user to explicitly choose beacon.anonymous
       // vs using an implied default so effectively this is binding
       // all values to their server-defaults (as client-side defaults dont really work)
+      // TODO(dcramer): we need to rethink this logic as doing multiple "is this value actually set"
+      // is problematic
       if (
-        option.value !== undefined &&
+        option.value !== undefined && option.value !== "" && option.value !== null &&
```

Comment:
So `false` and `0` are the only valid falsey values?

Reply:
Yeah I'm not sure - Im not a big fan of the way this works (the install wizard options, forms used here), but its very hard to come up with a "correct" approach that could be achieved in an afternoon.

Reply:
Specifically this behavior happened with mail.username and mail.password - those will no longer force the install wizard to prompt, it was just something I caught in parallel.